src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend class="legend">
+          Personal information
+        </legend>
+        <label class="label">
+          Surname:
+          <input
+            class="input"
+            type="text"
+            name="surname"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            class="input"
+            type="text"
+            name="name"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            class="input"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="age"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            class="input"
+            type="date"
+            name="birthDate"
+          >
+        </label>
+        <label class="label">
+          I accept all term of the agreement
+          <input
+            class="input"
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">
+          Registration
+        </legend>
+        <label class="label">
+          E-mail:
+          <input
+            class="input"
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            class="input"
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">
+          An interesting facts about you!
+        </legend>
+        <label class="label">
+          Do you love cats?
+          <input
+            class="input"
+            type="radio"
+            value="Yes"
+            name="cats"
+          >
+          Yes
+          <input
+            class="input"
+            type="radio"
+            value="No"
+            name="cats"
+          >
+          No
+        </label>
```

Comment:
You need 2 separate labels, because now I can't press to 'NO'

Reply:
My bad... Changing...